partiql-parser/src/main/antlr/Common.g4

```diff
@@ -0,0 +1,83 @@
+parser grammar Common;
```

Comment:
Personally, not a fan of splitting the files apart since we lose a lot of the git history associated w/ the old file. Though I don't know the best practice for ANTLR here. I've seen other open-source libraries like Presto just use one parser file and one lexer file.

Reply:
Ack. For the purposes of avoiding rebase chaos with the recent DDL addition, I'm going to merge all my current commits and move DML back into the single ANTLR file. All other commits will address your other comments.